AENS.md

```diff
@@ -66,19 +65,18 @@ could allow us an easier path for future update to the fee structures.
 
 Every entry in the `.test` namespace pays the same amount of fees.
 
+The new mechanism planted in Lima hard fork introduces auctions.
+We propose very simplistic scheme. The claim action becomes claim attempt. It can be followed by another claim from an account different than one set in preclaim for given name. There is finite amount of time when the follow up claim is allowed. This time is expressed in height delta computed from function of length of the name. In practice it means that the shorter the name, the longer we wait for another claim attempt. This should work as bidding. Furthermore, the initial fee is also a value of the function of a name length. Each next bid has to be higher by `X` tokens. Finally, we set a protocol protected length of name that is not subject to bidding and will be claimed immediately after confirming claim transaction. All, functions, base fee, free length value and price progression may be subject to changes with governance mechanism. Non-bidding path of the name claim is purposed for development or testing. Initial non-bidding length will be 32.
+
 Each entry has a fixed expiration date on claim after which the entry should
 transition into the `revoked` state. Once it reaches this state, the name
 will be released, i.e. transition into the `unclaimed` state, after which
 it could be claimed again.
 
 To prevent the entry from expiring a user can, at any point before reaching
 the expiration date, update the entry, which pushes the expiration date
-into the future by the time delta of the registration time and the update
-time, e.g. if a user posts an update one week after claiming their name,
-the expiration date gets pushed one week further into the future.
-The main motivation of this expiration date is to prevent situations where
-the private keys, which control the name, are lost making the name unusable
-as well.
+into the future by at most the maximum rent time e.g. if a user posts an update one week after claiming their name, the expiration date gets pushed for a full term from the previous expiration date. Follow up updates will have no effect, until we enter the new rent period. We also introduce rent payment for each update. The rent is locked in lock account and the rent fee is computed based on name length by function defined in governance.
```

Comment:
So this means that I can renew my name entry only after it entered the expiration period? Why?

Reply:
No, probably I need to re-phrase it.

Intention is following:
1. be similar to DNS re domain management
2. be able to renew at most for the next full term. No matter if you are at the beginning or the end of current term. Basically, the expiration update window is extended to full rent term. Also it limits rent time to one full next term. This is to avoid squating. Extending time window is for UX